js/index.js

```diff
@@ -32,4 +32,21 @@ document.addEventListener('DOMContentLoaded', e => {
     document.querySelector('.intro-background').addEventListener('transitionend', function() {this.style.display = 'none';});
     Particles.init(config);
   });
-});
\ No newline at end of file
+
+  document.querySelector('.mute-audio').addEventListener('click', e => {
+    // mute/unmute toggle
+    var element = document.querySelector('.mute-audio');
+    if(isAudioPlaying(audio)) {
+      audio.pause();
+      element.textContent = "Unmute"      
+    } else {
+      audio.play();
+      element.textContent = "Mute"            
+    }
+  });  
+
+});
+
+function isAudioPlaying(audioTrack) {
+  return !audioTrack.paused; 
```

Comment:
Since it is a single flag check, it does not need to be encapsulated in a function.

Reply:
Incorporated.